Documentation/operations/upgrade.rst

```diff
@@ -326,6 +326,9 @@ New Options
   for EC2 instances (and existing ENIs). EC2 Instances tags is in the form of k1=v1,k2=v2
   (multiple k/v pairs can also be passed by repeating the CLI flag")
   Use ``instanceTagsFilter`` in Helm chart.
+* ``nodes-gc-interval``: This option was marked as deprecated and has no effect
+  in 1.11. Cilium Node Garbage collector is added back in 1.12, so this flag
+  can be moved out of deprecated list.
```

Comment:
This option used to exist for kvstore GC. Now we are using it to implement K8s GC, so we should point that out.